tests/components/transmission/test_config_flow.py

```diff
@@ -0,0 +1,195 @@
+"""Tests for Met.no config flow."""
+from datetime import timedelta
+from unittest.mock import patch
+
+import pytest
+from transmissionrpc.error import TransmissionError
+
+from homeassistant import data_entry_flow
+from homeassistant.components.transmission import config_flow
+from homeassistant.components.transmission.const import (
+    DEFAULT_NAME,
+    DEFAULT_PORT,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+)
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
+
+from tests.common import MockConfigEntry
+
+NAME = "Transmission"
+HOST = "192.168.1.100"
+USERNAME = "username"
+PASSWORD = "password"
+PORT = 9091
+SCAN_INTERVAL = 10
+
+
+@pytest.fixture(name="api")
+def mock_transmission_api():
+    """Mock an api."""
+    with patch("transmissionrpc.Client", return_value=True):
```

Comment:
Remove the return value. We should just set a mock instance as return value, which is the default.

Reply:
Can you please suggest the exact code as i am new to tests

Reply:
```py
with patch("transmissionrpc.Client"):
    yield
```